launch/schunk_neck.launch

```diff
@@ -0,0 +1,21 @@
+<launch>
+  <!-- Load the controllers -->
+  <rosparam command="load" ns="schunk_robot/" file="$(find herb_launch)/config/schunk_controllers.yaml"/>
+
+  <node name="schunk_pw70_ros" pkg="schunk_pw70_ros" type="schunk_robot" ns="schunk_robot/" output="screen"/>
+
+  <!-- TODO timeout? -->
```

Comment:
Can you clarify this TODO further in the comment if it's still valid?

Reply:
This is actually copied from `ros_control.launch`, so I'm not really sure what the TODO is. I think we should resolve both at the same time, though.